### PR TITLE
Update pull request template

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

like [kubernetes ](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md) does, put pr kink in the comment.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -6,22 +6,28 @@
...
+/kind bug
+/kind cleanup
+/kind documentation
+/kind feature
+/kind test
+/kind design
+
+Optionally add one or more of the following kinds if applicable:
+/kind api-change
+/kind design
```
Comment (fisherxu) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: remove the duplicate line :)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -6,22 +6,27 @@
...
+/kind cleanup
+/kind documentation
+/kind feature
+/kind test
+/kind design
+
+Optionally add one or more of the following kinds if applicable:
+/kind api-change
+/kind failing-test
+/kind regression
```
Comment (fisherxu) [inline .github/PULL_REQUEST_TEMPLATE.md:22]: now we still have no the `regression` kind label, need upgrade the prow later.

Comment (daixiang0) [reply .github/PULL_REQUEST_TEMPLATE.md:22]: Get it. Tracked as #2104

**Pull request merged**